client/src/Components/ProductCard/ProductCard.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import Product from "../../Models/Product"
+
+import "./ProductCard.css"
+
+interface ProductCardProps {
+    product: Product
+}
+
+const ProductCard: React.FC<ProductCardProps> = ({ product }) => {
+    return (
+        <div className="product-card">
+            <ul>
```

Comment:
The way you currently have this set up you are creating a new unordered list with a single list item for every product displayed. Move the `<ul>` to the ProductList component. Wrap the products.map in the `<ul>`, then just have an `<li>` here in the ProductCard. That will give you a single unordered list with each product as a list item inside of that unordered list. 

I would also just replace the ` <div className="product-card">` with `<li className="product-card">`. The more you nest elements inside of other elements, the harder it can be to fix styling.